.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           command: "cd tdrs-backend; docker-compose down "
       - run:
           name:  "Test: Execute Python Unit Tests"
-          command: "cd tdrs-backend; docker-compose run web sh -c \"python manage.py test\""
+          command: "cd tdrs-backend; docker-compose run web sh -c \"pytest\""
       - run:
           name:  "Test: Execute Python Linting Test"
           command: "cd tdrs-backend; docker-compose run --rm web bash -c \"flake8 .\""
```

Comment:
We'll be using `black` to format our code  as part of our local builds /code editing so it can be properly formatted before we commit. We could use `black --check .` to see if there was anything we missed but flake8 should catch it as well without needing to write a fancy exit code script to catch anything found by black.